### ISSUE TITLE
[Meta-issue] Release 0.47.0

### ISSUE THREAD

**Issue by connortann** [shap/shap#3841]
Before making release:

- [x] Fix conda-forge release pipeline, and ideally release 0.46 on conda-forge: #3688
   - https://github.com/conda-forge/shap-feedstock/pull/85
   - #3825
- [x] Implement all scheduled deprecations, as per: #3507
  - #3858
  - #3739
  - #2624
  - #3856
- #3979

Then, regular release checklist:

- [x] Dry-run cibuildwheel & test
- [x] Make GitHub release & tag
- [x] Confirm PyPI wheels published
- [x] Conda forge published

**Comment by CloseChoice** [comment]
I would like get https://github.com/shap/shap/pull/3862 in, so that we might get the conda builds to pass

**Comment by connortann** [comment]
Hi @shap/shap-maintainers ,

I'd think we're ready to make a release now. I'll hold off for a couple of days in case anyone has any objections or suggestions before we go ahead. In the meantime, I'll check the cibuildwheel workflow passes.

There has been a larger gap than usual between this release and the previous one, I think due to a combination of factors: we collectively haven't had that much time for SHAP maintenance, and the conda-forge workflow was held up for a while. However, I think we should be able to resume a semi-regular release cycle now that the conda-forge workflow is passing again 👍 

@CloseChoice  @thatlittleboy

**Comment by CloseChoice** [comment]
@connortann I'll push for a release, at best this week. There is one PR I'd like to get in (https://github.com/shap/shap/pull/3944) then I think we're good.

Any objections? @connortann would really appreciate if you could look over the draft release I'll create within the next days.

**Comment by connortann** [comment]
Hi @CloseChoice , that sounds like a good plan. I'm afraid I haven't had any capacity this week but early next week could be good. Let me know when you have a draft release!

**Comment by CloseChoice** [comment]
@connortann: here you go, happy for you input https://github.com/shap/shap/releases/tag/untagged-c506f88221ba039cd92b

**Comment by connortann** [comment]
@CloseChoice that looks overall good to me. Lots of work that I think the community can be proud of!

One suggestion (and lesson from previous releases) is that we should give more detail on the breaking changes, how they affect users and what is required to update (if anything). In this case, it's #3739.

Currently it's described as "Deprecate the legacy bar plot". May I suggest rewording that as, "Add deprecation warning to legacy bar plot", just to avoid confusion with "deprecation" meaning removing the functionality?

Are you planning to go ahead and trigger the release? For what it's worth I think it that would be a good idea, as I've done it in the past and we want to ensure there are multiple maintainers able to make releases.

**Comment by CloseChoice** [comment]
Thanks for taking a look, I updated the description. I'll go ahead and make the release within the remainder of the week.

Yes, I was also positively surprised by all the work and the amount of new contributors.

**Comment by CloseChoice** [comment]
The release is through, closing.
